build/generate-cli-docs/generate.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cobra/doc"
+
+	"github.com/open-policy-agent/opa/cmd"
+)
+
+var fileHeader = `---
+title: CLI
+kind: documentation
+weight: 90
+restrictedtoc: true
+---
+
+The OPA executable provides the following commands.
```

Comment:
There seems to be a big space between the title and the first text line. This seems to be the case for other releases too. I must have failed to notice that before but was obvious from looking at the preview for this page.